src/smart-components/group/add-group/schema.js

```diff
@@ -31,8 +28,7 @@ export default {
             {
               component: componentTypes.TEXTAREA,
               name: 'group-description',
-              type: 'text',
-              label: 'Group description',
+              hideField: true,
```

Comment:
Is this supposed to be hidden?

Reply:
@Hyperkid123 yeah, it is. The field is there because of validation for wizard buttons. Visible input is in the step custom component. I am not sure if this is the best solution but it is the only one that worked.